<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="RVSKCH90LT" name="com.rvsappiumautomation.testcases.FirstPage" tests="3" failures="0" timestamp="13 Jan 2020 12:04:42 GMT" time="54.486" errors="2">
  <testcase name="testCase002" time="27.613" classname="com.rvsappiumautomation.testcases.FirstPage">
    <error type="com.rvsappiumautomation.exception.ExceptionHandler" message="This test is failed because of the Exception in com.rvsappiumautomation.testcases.FirstPage.testCase002()
The reason of this exception is: Expected condition failed: waiting for visibility of element located by By.xpath: //android.widget.Button[@resource-id=&#039;com.android.packageinstaller:id/permission_allow_button&#039;] (tried for 25 second(s) with 500 milliseconds interval)">
      <![CDATA[com.rvsappiumautomation.exception.ExceptionHandler: This test is failed because of the Exception in com.rvsappiumautomation.testcases.FirstPage.testCase002()
The reason of this exception is: Expected condition failed: waiting for visibility of element located by By.xpath: //android.widget.Button[@resource-id='com.android.packageinstaller:id/permission_allow_button'] (tried for 25 second(s) with 500 milliseconds interval)
	at com.rvsappiumautomation.testcases.FirstPage.testCase002(FirstPage.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- testCase002 -->
  <testcase name="testCase001" time="1.380" classname="com.rvsappiumautomation.testcases.FirstPage"/>
  <testcase name="testCase003" time="25.493" classname="com.rvsappiumautomation.testcases.FirstPage">
    <error type="com.rvsappiumautomation.exception.ExceptionHandler" message="This test is failed because of the Exception in com.rvsappiumautomation.testcases.FirstPage.testCase003()
The reason of this exception is: Expected condition failed: waiting for visibility of element located by By.xpath: //android.widget.Button[@resource-id=&#039;com.android.packageinstaller:id/permission_allow_button&#039;] (tried for 25 second(s) with 500 milliseconds interval)">
      <![CDATA[com.rvsappiumautomation.exception.ExceptionHandler: This test is failed because of the Exception in com.rvsappiumautomation.testcases.FirstPage.testCase003()
The reason of this exception is: Expected condition failed: waiting for visibility of element located by By.xpath: //android.widget.Button[@resource-id='com.android.packageinstaller:id/permission_allow_button'] (tried for 25 second(s) with 500 milliseconds interval)
	at com.rvsappiumautomation.testcases.FirstPage.testCase003(FirstPage.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- testCase003 -->
</testsuite> <!-- com.rvsappiumautomation.testcases.FirstPage -->
